### DISCUSSION TITLE
Allow more normalize methods in getGroupBW

### DISCUSSION THREAD

**Discussion by yimingsun12138** [GreenleafLab/ArchR#1457] [Feature Requests]
Hello, thanks for your great work of ArchR, it really helps a lot.

I notice that `getGroupBW` function only allows several normalization methods, as you can see in the code:
`normMethods <- c("None", "ReadsInTSS", "nCells", "ReadsInPromoter", 
        "nFrags")
    if (tolower(normMethod) %ni% tolower(normMethods)) {
        stop(paste0("normMethod (", normMethod, ") not in supported normMethods : ", 
            paste0(normMethods, collapse = ", ")))}`

If we treat peak region as the real signal indicating open chromatin, I think reads in peak is also a good way to normalize the sequencing depth while controlling the sequencing quality. So, have you ever tried to do the normalization by ReadsInPeaks and can you allow more columns in `CellColdata` to do the work? I read all the code and I think this can be easily achieved.

Anyway, thank you for your great work!

**Comment by rcorces** [comment]
ReadsInPeaks really isnt going to change this for the better. We left out ReadsInPeaks to prevent end users from doing something that doesnt make sense (for example, using a peak set that is created outside of ArchR and normalizing by peaks that are not well balanced across the cell types in the ArchRProject OR adding a peak set but forgetting to overwrite the peak matrix which would recalculate ReadsInPeaks). ReadsInTSS is a very stable way of doing the normalization which is why it is the default. ReadsInPeaks, on the other hand, depends on the peak set which is different across projects.

The problem with allowing arbitrary columns in `cellColData` to use for normalization is that different inputs would need to be handled differently and its hard to prevent end users from breaking the function with unexpected inputs. for ex:

https://github.com/GreenleafLab/ArchR/blob/21099d6ed966f6088b7b49ebbb9cca07486e8875/R/GroupExport.R#L356-L363

I can enable ReadsInPeaks normalization but I'm not very inclined to do so because I dont think it adds to the functionality of ArchR. If you have data to support otherwise, I'm happy to consider it.
